doc/generate_cet.py

```diff
@@ -3,7 +3,121 @@
 import pathlib
 
 
+def provide_ignored():
+    ignored_attributes = ["EFDB_Name", "Description"]
+    ignored_fields = ["Description", "Alias", "Device", "Property", "Action", "Value", "Subsystem", "Version", "Author", "Explanation"]
+    return ignored_attributes, ignored_fields
+
+def add_generics(cf, subsystem, set_name, has_generics, has_specific):
+    ignored_attributes, ignored_fields = provide_ignored()
+    if has_generics is True:
+        gen_tree = ET.parse("../sal_interfaces/SALGenerics.xml")
+        gen_root = gen_tree.getroot()
+        for gen_set in gen_root:
+            gen_set[:] = sorted(gen_set, key=lambda child: (child.tag, child.find("EFDB_Topic").text.split("_")[-1] if child.find("EFDB_Topic") is not None else child.tag))
```

Comment:
I'm puzzled why you sort the topics here yet somehow manage to interleave generic and non-generic topics in alphabetical order. A code comment explaining what's going on would help future maintainers of the code.

Reply:
Still want a code comment